### PR TITLE
Create Button Component

### PR DESCRIPTION
**Pull request by hashemt**
Relates #25

### CODE REVIEW

File: client/src/components/common/Button/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = ({ className, buttonFunction, value }) => {
+  return (
+    <button className={className} onClick={buttonFunction} type="button">
```
Comment (AbdallahAmmar96) [inline client/src/components/common/Button/index.js]: Add a class name for the button 
```suggestion
    <button className={`btn ${className}`} onClick={buttonFunction} type="button">
```

File: client/src/components/common/Button/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
```
Comment (AbdallahAmmar96) [inline client/src/components/common/Button/index.js]: Add default style for the button

File: client/src/components/common/Button/index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+const Button = ({ className, buttonFunction, value }) => {
```
Comment (AbdallahAmmar96) [inline client/src/components/common/Button/index.js]: Instead of value use children

File: client/src/components/common/Button/style.css [hunk 4]
```diff
@@ -0,0 +1,8 @@
+.btn{
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
```
Comment (mossa-Sammer) [inline client/src/components/common/Button/style.css]:
```suggestion
    display: block;
```

File: client/src/components/common/Button/style.css [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+    cursor: pointer;
+    font-size: 16px;
+    border-radius: 10px;
```
Comment (mossa-Sammer) [inline client/src/components/common/Button/style.css]:
```suggestion
    border-radius: 5px;
```

Comment (fares98) [inline client/src/components/common/Button/style.css]: Add the font family and the main color for the button

File: client/src/components/common/Button/style.css [hunk 6]
```diff
@@ -0,0 +1,8 @@
+.btn{
+    text-align: center;
+    text-decoration: none;
+    display: block;
+    cursor: pointer;
+    font-size: 16px;
+    border-radius: 3px;
```
Comment (AbdallahAmmar96) [inline client/src/components/common/Button/style.css]:
```suggestion
      text-transform: uppercase;
  border: none;
  border-radius: 5px;
  color: white;
  letter-spacing: 1.5px;
  font-weight: 600;
  font-size: 20px;
  padding: 5px 10px;
  cursor: pointer;
```

File: client/src/components/common/Button/style.css [hunk 7]
```diff
@@ -0,0 +1,13 @@
+.btn{
+    text-align: center;
+    text-decoration: none;
+    display: block;
```
Comment (fares98) [inline client/src/components/common/Button/style.css]: delete this line

**Pull request merged**